deploy/test-environments/delete_env.sh

```diff
@@ -17,6 +17,8 @@ AWS_REGION="eu-west-1" # Add your desired default AWS region here
 DELETED_ENVS=()
 FAILED_ENVS=()
 
+: "${TF_VAR_qa_ec_api_key:?Please set TF_VAR_qa_ec_api_key with an Elastic QA Cloud API Key}"
```

Comment:
@Omolola-Akinleye I'm unsure if this is the appropriate location to request the `TF_VAR_qa_ec_api_key`. Since I intend to delete a production environment, the `TF_VAR_qa_ec_api_key` isn't necessary. If I define it here, both keys would be required.